tests/usage_test.py

```diff
@@ -62,7 +65,7 @@ def test_no_args(self):
 
         self.assertEqual(stdout.getvalue(), EXPECTED)
 
-    def test_valid_args_but_no_filepath(self):
+    '''def test_valid_args_but_no_filepath(self):
```

Comment:
filepath option removed

Reply:
👍  You can change this to match something along the lines of `targets argument is required`